x/staking/keeper/params.go

```diff
@@ -39,13 +39,6 @@ func (k Keeper) BondDenom(ctx sdk.Context) (res string) {
 	return
 }
 
-// PowerReduction - is the amount of staking tokens required for 1 unit of consensus-engine power
-// governance can update it on a running chain
-func (k Keeper) PowerReduction(ctx sdk.Context) (res sdk.Int) {
-	k.paramstore.Get(ctx, types.KeyPowerReduction, &res)
```

Comment:
we could use the `sdk.DefaultPowerReduction` here, and reduce number of changes, and prepare for the correct implementation.

Reply:
good idea, will make this change